node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentContextImpl.java

```diff
@@ -138,9 +146,11 @@ public void log(Logger logger, Level level, String message, Throwable throwable)
         private Path pathToContainerStorage;
         private Path pathToVespaHome;
         private String vespaUser;
-        
-        public Builder(String hostname) {
+        private Integer vespaUserIdOnHost;
+
+        public Builder(String hostname, int vespaUserIdOnHost) {
```

Comment:
Don't add this constructor, the idea of this builder is to create a context as easily as possible for the tests and the other fields that the test doesn't really care about should just take whatever reasonable value.

Reply:
In that case, what ID should be returned by default? Hard code 1000?

Reply:
Doesn't matter, the entire point is that most tests will not test something that uses the value, or even if they do, they do not assert that value. For the tests this is important, they can explicitly set it.

Reply:
The class is documented to be used only for tests. Fixed.